bin/hbase

```diff
@@ -656,6 +657,24 @@ elif [ "$COMMAND" = "pre-upgrade" ] ; then
   CLASS='org.apache.hadoop.hbase.tool.PreUpgradeValidator'
 elif [ "$COMMAND" = "completebulkload" ] ; then
   CLASS='org.apache.hadoop.hbase.tool.BulkLoadHFilesTool'
+elif [ "$COMMAND" = "hbtop" ] ; then
+  CLASS='org.apache.hadoop.hbase.hbtop.HBTop'
```

Comment:
What's with all this gymnastics building a custom class path? Looks like a land mine waiting to trip up someone who wants to upgrade our dependencies.

Reply:
Any command that needs something beyond our basics (either client or normal service process) has to whitelist the extras it needs.

hbtop largely just needs the shaded client jar, but it has some third-party stuff it uses directly, hence the extra add on.

I think this is the best of not great options given how we currently do packaging.

If you want to discuss improvements to how we do command packaging it's something I'm interested in. A DISCUSS thread or an umbrella jira would probably be a better venue for that